chef/cookbooks/cinder/templates/default/cinder.conf.erb

```diff
@@ -740,7 +740,10 @@ nova_api_insecure=<%= @nova_api_insecure %>
 # the direct_url.  Currently supported schemes: [file]. (list
 # value)
 #allowed_direct_url_schemes=
-
+<% if @show_storage_location -%>
+### Enable file and cinder if available
+allowed_direct_url_schemes=rbd
+<% end -%>
 
 #
 # Options defined in cinder.image.image_utils
```

Comment:
In the other PR, we have https://github.com/crowbar/crowbar-openstack/pull/110/files#diff-4e35a687f5956e18c68d43a410d78e78R1023 (`image_upload_use_cinder_backend = True` and `image_upload_use_internal_tenant = False`). This is not available in juno?